VirtusLab-Open-Source/strapi-plugin-navigation
# feat: live updates of navigation items

## Ticket

https://github.com/VirtusLab-Open-Source/strapi-plugin-navigation/issues/152

## Summary

What does this PR do/solve? 

- live updates of navigation item form fields

## Test Plan

- start the app
- go to admin panel
- add/edit navigation item
- change related item
- if item is configured to update navigation item form fields, then updates should be applied
- auto updates can be disabled
